cEP-0005.md

```diff
@@ -124,6 +124,15 @@ class RedundancyBear(LocalBear):
             yield Result.from_values(aspect=aspect, ...)
 ```
 
+## Aspects
+An aspect should just talk about a property of something, not qualifying it. Don't name it TooLong but Length.
+Then give it settings, e.g. Metadata.CommitMessage.Shortlog.Length can have a setting max_shortlog_length.
+
+## Settings
```

Comment:
@EverWinter23 As @Techievena mentioned it's really important to rename *settings* to *tastes*, since the latter is the new official term.